_press/history-of-18F.md

```diff
@@ -6,6 +6,6 @@ layout: bare
 
 * GSA launched 18F on March 19, 2014. The name is an abbreviation for GSA’s address — 1800 F St.
 
-* The 18F team started with 15 full time staff. Our team has grown to over 100 as of January 2015, as 18F’s product and client work has increased.
+* The 18F team started with 15 full time staff. Staffing has grown to over 180 as of May 2016, as 18F’s product and client work has increased.
 
-* Our goal was (and is) to transform the way the U.S. government approaches problems. You can [read more about our first year on our blog](https://18f.gsa.gov/2015/03/20/one-year-in-and-looking-forward/).
+* Our goal was (and is) to transform the way the U.S. government approaches problems. Read more about our evolution in the blog posts from our [first](https://18f.gsa.gov/2015/03/20/one-year-in-and-looking-forward/) and [second](https://18f.gsa.gov/tags/second-anniversary/) anniversaries.
```

Comment:
@gboone side request -- create a `first-anniversary` tag for the blog posts we published last year and then change the link here to the tag instead of the single post?